packages/cypress/package.json

```diff
@@ -17,7 +17,6 @@
     "*.d.ts"
   ],
   "scripts": {
-    "postinstall": "node ./first-run.js",
```

Comment:
this is a breaking change, so it deserves a major bump in the relevant packages - and we might want to take a look if there are any other breaking changes in those packages that we'd like to batch together with this

Reply:
nothing comes to mind. will defer to you.